### DISCUSSION TITLE
Which key for python next/prev reference seem wrong display

### DISCUSSION THREAD

**Discussion by tranbinh1604** [LazyVim/LazyVim#3683] [Q&A]
Hi, I think its wrong display, only happen when in `python` files (open python file > press` ]` or `[`, then which key show up as below. Im using default configs for which key.

![SS 2024-06-16 at 11 05 10](https://github.com/LazyVim/LazyVim/assets/162952043/cb6f90b3-81a6-4842-bc56-4b242225f4ba)

![CleanShot 2024-06-16 at 11 26 40](https://github.com/LazyVim/LazyVim/assets/162952043/01804d27-1fc0-447c-a806-bb9f29c6599e)

**Comment by dpetka2001** [comment]
It still shows correctly `Next/Prev Reference` in your image. What you're referring is the mappings `[]/][`. See `:h []` for more info on what they do.

**Reply by tranbinh1604** [reply]
not the key `[` or `]`, I mean the description right next the key

`<C-U>call <SNR>44_Python_jump)('n',   `

but if this is the correct desc, then what does it mean?

**Reply by dpetka2001** [reply]
I don't know where this comes from, but it shows the correct description for `Next/Previous Reference`. This is something else than highlight references. You can see in the second image a few lines below what you have in red color that it shows the `Next Reference` in the description.

**Reply by dpetka2001** [reply]
Apparently this comes from Python ftplugin as you can see [here](https://github.com/neovim/neovim/blob/615e859a0e0a6410dccc095693350e0ec2a280de/runtime/ftplugin/python.vim#L49-L85).

**Reply by tranbinh1604** [reply]
> I don't know where this comes from, but it shows the correct description for `Next/Previous Reference`. This is something else than highlight references. You can see in the second image a few lines below what you have in red color that it shows the `Next Reference` in the description.

Yes. Only when you are in a python file, the rest are correct, only the red block description quite weird.

**Reply by dpetka2001** [reply]
It's from Neovim runtime Python ftplugin. See the link in my previous comment.
